Arrays.java

```diff
@@ -0,0 +1,9 @@
+package com.edocti.jintro.lab01;
+	public class Arrays {
```

Comment:
You don't need to indent the class. Classes (except inner classes, which we didn't study yet) are placed on the "first level", like package and import statements